Sources/MeiliSearch/Model/Task.swift

```diff
@@ -21,7 +21,20 @@ public struct Task: Codable, Equatable {
   public let status: Task.Status
 
   /// Type of the task.
-  public let type: String
+  //public let type: String
+    public enum Type:Codable {
```

Comment:
Also, there is some GithubAction annotations, please fix them:

<img width="883" alt="image" src="https://user-images.githubusercontent.com/4116980/227535375-56d25555-9cec-4757-85e8-b5e5b0c1e1b2.png">